test/hotspot/gtest/code/test_codestrings.cpp

```diff
@@ -45,12 +45,17 @@ static const char* replace_addr_expr(const char* str)
     std::basic_string<char> tmp1 = std::regex_replace(str, std::regex("0x[0-9a-fA-F]+"), "<addr>");
     // Padding: aarch64
     std::basic_string<char> tmp2 = std::regex_replace(tmp1, std::regex("\\s+<addr>:\\s+\\.inst\\t<addr> ; undefined"), "");
-    std::basic_string<char> tmp3 = std::regex_replace(tmp2, std::regex("\\s+<addr>:\\s+udf\\t#0"), "");
+    std::basic_string<char> tmp3 = std::regex_replace(tmp2, std::regex(",\\s+.*<addr>"), "");
+    std::basic_string<char> tmp4 = std::regex_replace(tmp3, std::regex(", <addr> = .*"), "");
+    std::basic_string<char> tmp5 = std::regex_replace(tmp4, std::regex("b[\\\\t\\s]+.*"), "b <addr>");
+    std::basic_string<char> tmp6 = std::regex_replace(tmp5, std::regex("adrp[\\\\t\\s]+([wx][0-9]+).*"), "adrp $1 = <addr>");
+    std::basic_string<char> tmp7 = std::regex_replace(tmp6, std::regex(", \\w+::\\w+.*"), "");
+    std::basic_string<char> tmp8 = std::regex_replace(tmp7, std::regex("\\s+<addr>:\\s+udf\\t#0"), "");
```

Comment:
It would help this reviewer if you added some commentary that explains what's going on. An example of the AArch64 output would be nice.

Reply:
Okey, I will add some commentary later.